src/tests/ftest/soak/utils.py

```diff
@@ -1150,17 +1150,15 @@ def build_job_script(self, commands, job, nodesperjob):
     self.log.info("<<Build Script>> at %s", time.ctime())
     script_list = []
     # if additional cmds are needed in the batch script
-    prepend_cmds = [
-        "set -e",
-        "echo Job_Start_Time `date \\+\"%Y-%m-%d %T\"`",
-        "daos pool query {} ".format(self.pool[1].uuid),
-        "daos pool query {} ".format(self.pool[0].uuid)
-        ]
-    append_cmds = [
-        "daos pool query {} ".format(self.pool[1].uuid),
-        "daos pool query {} ".format(self.pool[0].uuid),
-        "echo Job_End_Time `date \\+\"%Y-%m-%d %T\"`"
-        ]
+    prepend_cmds = ["set -e",
+                    "echo Job_Start_Time `date \\+\"%Y-%m-%d %T\"`",
+                    "daos pool query {} ".format(self.pool[1].uuid),
+                    "daos pool query {} ".format(self.pool[0].uuid)
+                    ]
+    append_cmds = ["daos pool query {} ".format(self.pool[1].uuid),
+                   "daos pool query {} ".format(self.pool[0].uuid),
+                   "echo Job_End_Time `date \\+\"%Y-%m-%d %T\"`"
+                   ]
```

Comment:
Is this really our preferred style? I don't like this

Reply:
Flake is checking that tha ] is aligned just to the right of the ], I suspect it would also allow it if the ] was at the end of the previous line however.

Reply:
I more so meant that I prefer:
```
    prepend_cmds = [
        "set -e",
        "echo Job_Start_Time `date \\+\"%Y-%m-%d %T\"`",
        "daos pool query {} ".format(self.pool[1].uuid),
        "daos pool query {} ".format(self.pool[0].uuid)]
```
OR
```
    prepend_cmds = [
        "set -e",
        "echo Job_Start_Time `date \\+\"%Y-%m-%d %T\"`",
        "daos pool query {} ".format(self.pool[1].uuid),
        "daos pool query {} ".format(self.pool[0].uuid)
    ]
```

I despise having to indent something that isn't aligned with tab width. But we do that in all of our Python and C code, so I suspect I've already lost that battle. To my point: think about what happens if the length of the variable name `prepend_cmds` changes. You have to reformat the whole list! It's also a silly notion that indentation is tied to the length of a variable name or function call. I digress

Reply:
The later is what we had before (which I personally don't like) but I think the former would work.  The whole flake checking is new, what I did for this PR was ran "flake8 --statistics" and fixed/resolved any errors that had fewer than 10 or so warnings and enabled the check.  I'll try the former from above, and if that doesn't work then I'll back out this part and leave that check disabled.

> To my point: think about what happens if the length of the variable name prepend_cmds changes. You have to reformat the whole list! It's also a silly notion that indentation is tied to the length of a variable name or function call. I digress
The point of having auto-formatting is that doesn't matter any more, yes the code gets re-indented but it happens automatically, and if everybody does it then whilst everybody does it the resulting code is the same so there are no merge conflicts and the code having a consistent style really does make it easier to read.

Reply:
In the end, I think consistency is most important.